docs/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+You can refer to [Kubernete's community documentation] on writing good commit messages, which provides good tips and ideas.
+
+Some PRs addess specific issues. Please, refer to the [CONTRIBUTING] documentation for more
+information on how to link a PR to an existing issue.
+
+[Kubernete's community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
+[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md
+
+# What environments does this code impact?
+
+- [ ] Cloud
+- [ ] Onprem
+- [x] None
+
+
+# How was this code tested?
+
+Please, select one or more if needed:
+
+- [ ] Local unit-tests
+- [ ] Local Kupe API subsystem tests
+- [ ] assisted-test-infra environment
+- [ ] dev-scripts environment
+- [ ] Reviewer's test appreciated
+- [ ] Waiting for CI to do a full test run
+- [x] No tests needed
+
+It's recommended to take a few extra minutes to provide more information about
+how this code was tested. Here are some questions that may be worth answering:
+
+- Should this PR be tested by the reviewer?
+- Is this PR relying on CI for an e2e test run?
+- Should this PR be tested in a specific environment?
+- Any logs, screenshots, etc that can help with the review process?
+
+
+# Assignees
+
+Please, add one or two reviewers that could help review this PR.
+
+/assign @
+/assign @
```

Comment:
This part is automatic

Reply:
I would like for contributors to think about who could help with their reviews. The reviewers picked are not always the right/best ones.

I think having this, at least until we clean-up the repo a bit and create proper OWNERS files for every subpath, is a good exercise.

Reply:
@YuviGold We already had a discussion about this a while ago. Automatic assignments create too much noise - the bot can't take into consideration the areas of expertise of a developer, so sometimes the assigned reviewer(s) can't do a meaningful review, or need a lot of time just to understand the context. 

If it's not possible to disable the automatic assignment, we at least should carefully /cc and /uncc, IMO.